runtime/common/src/crowdloan.rs

```diff
@@ -1565,7 +1669,7 @@ mod benchmarking {
 			let caller: T::AccountId = whitelisted_caller();
 			let contribution = T::MinContribution::get();
 			T::Currency::make_free_balance_be(&caller, BalanceOf::<T>::max_value());
-		}: _(RawOrigin::Signed(caller.clone()), fund_index, contribution)
+		}: _(RawOrigin::Signed(caller.clone()), fund_index, contribution, None)
```

Comment:
same here

Reply:
What is the best way for me to generate a signature here for benchmarking purpose? Can't find similar examples.

Reply:
Probably you can use the crypto libraries to sign the message you expect, and then verify the signature all at once.

If you want to make the benchmarking a bit more simple, you could make a benchmark that only does signature verification on its own, and then add that to all the extrinsics which have some signature verification part.

Reply:
The problem is the benchmark will only work if the runtime using this pallet is configured to use the exact signing algorithm.

I guess not a major block here. So the next question is what signature type will kusama/polkadot use? `MultiSignature`?

Reply:
I think SR25519 and ED25519 will be all you need here.